commercetools-models/src/main/java/io/sphere/sdk/projects/CartClassification.java

```diff
@@ -0,0 +1,35 @@
+package io.sphere.sdk.projects;
+
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import io.sphere.sdk.annotations.ResourceValue;
+import io.sphere.sdk.models.LocalizedString;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@ResourceValue
+@JsonDeserialize(as = CartClassificationImpl.class)
+public interface CartClassification extends ShippingRateInputType{
+
+    @Override
+    default String getType(){
+        return "CartClassification";
+    }
+
+    Set<CartClassificationEntry> getValues();
+
+    static CartClassification of(final Map<String, LocalizedString> values){
+        final Set<CartClassificationEntry> cartClassificationEntries = values.entrySet().stream()
```

Comment:
Here I'm fine with having static factory method, because this type is used in our read and write. But another option would to use our `HasBuilder` annotation to generate a builder class instead. What do you think?

Reply:
Actually i think that making the separation between read write model is much better, since by the time the two models might get much different and get the backward compability difficult to keep.